packages/react/src/internal.tsx

```diff
@@ -1,6 +1,7 @@
 export * from './hooks/useAuth';
 export * from './hooks/useDataStore';
 export * from './hooks/useStorageURL';
+export { useNavigateAction } from './hooks/actions/useNavigateAction';
```

Comment:
Should we also export `UseNavigateActionOptions`?

Reply:
Yea, that would be great. Will update!